ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/shacl/entities/ShaclShape.java

```diff
@@ -0,0 +1,37 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.shacl.entities;
+
+import org.apache.jena.rdf.model.Model;
+
+import java.util.Objects;
+
+public class ShaclShape {
+	private final String collection;
+	private final Model model;
+
+	public ShaclShape(String collection, Model model) {
+		this.collection = collection;
+		this.model = model;
+	}
+
+	public String getCollection() {
+		return collection;
+	}
+
+	public Model getModel() {
+		return model;
+	}
+
+	@Override
+	public boolean equals(Object o) {
+		if (this == o)
+			return true;
+		if (!(o instanceof ShaclShape that))
+			return false;
+		return Objects.equals(collection, that.collection);
```

Comment:
why are we only comparing the collection and ignoring the model? I noticed that on Wouter his branch, the model is included as well.

Reply:
In spirit of DDD, ShaclShape is considered an entity and the collection is the entity's identity